deploy/charts/approver-policy/templates/webhook.yaml

```diff
@@ -49,7 +49,7 @@ webhooks:
       service:
         name: {{ include "cert-manager-approver-policy.name" . }}
         namespace: {{ .Release.Namespace | quote }}
-        path: /validate
+        path: /validate-policy-cert-manager-io-v1alpha1-certificaterequestpolicy
```

Comment:
How this path is generated seems to now be hardcoded in c/r https://github.com/kubernetes-sigs/controller-runtime/blob/v0.15.0-alpha.0/pkg/builder/webhook.go#L247-L250

Reply:
Hmm.. don't think it's a major issue/something we can really address easily, but this will create a breaking change at runtime too (requests will fail after this Webhook resource is updated, until the new version of the Deployment rolls out 😬)

Reply:
Yes, I agree. I think in majority of cases folks should not notice anything, but I will add a clear warning to the release notes.

I don't see a straightforward way how to keep the existing path unless we don't use their new webhook builder mechanism and I think that writing a standalone webhook just to prevent this path change would be an overkill and too much of a maintenance burden